python/lib/core/dmod/core/meta_data.py

```diff
@@ -698,6 +878,67 @@ def factory_init_from_restriction_collections(cls, data_format: DataFormat, **kw
                           continuous_restrictions=None if len(continuous) == 0 else continuous,
                           discrete_restrictions=None if len(discrete) == 0 else discrete)
 
+    @classmethod
+    def merge_domains(cls, d1: 'DataDomain', d2: 'DataDomain') -> 'DataDomain':
```

Comment:
Rename the variables to avoid unnecessary acronyms. Try `domain_one` and `domain_two`.

Reply:
I still think the short names are preferable in this situation.  I feel like the gist of what they are becomes clear right away from the function signature, with nothing really gained by spelling things out.  That said, it's difficult to separate myself from my current familiarity with it.  Any strong disagreement?